src/components/Listing/Listing.tsx

```diff
@@ -3,6 +3,7 @@ import cx from 'classnames';
 
 import ListingElement from './ListingElement';
 import ListingLink from './ListingLink';
+import StatisticCard from '../StatisticCard/StatisticCard';
```

Comment:
Hey @DipakShrestha18 :)

There is a very specific way in which card listings are put together to ensure they have the correct width and spacing etc. It doesn't look like this imported module `StatisticCard` is actually being used in here so it can be removed.

Take a look at at [ListingElement.tsx](https://github.com/wellcometrust/corporate-components/blob/develop/src/components/Listing/ListingElement/ListingElement.tsx) and look at how "ImageCard" has been handled. Let me know if you want to go through this step by step.